src/Sylius/Behat/Context/Ui/CheckoutContext.php

```diff
@@ -156,23 +165,100 @@ public function iSpecifyTheBillingAddressAs(AddressInterface $address)
     }
 
     /**
+     * @When /^I specified the shipping (address as "([^"]+)", "([^"]+)", "([^"]+)", "([^"]+)" for "([^"]+)")$/
+     */
+    public function iSpecifiedTheShippingAddress(AddressInterface $address)
+    {
+        $this->addressingPage->open();
+        $this->addressingPage->specifyShippingAddress($address);
+        $this->iProceedWithTheShippingStep();
+    }
+
+    /**
      * @When I choose the different billing address
      */
     public function iChooseTheDifferentBillingAddress()
     {
         $this->addressingPage->chooseDifferentBillingAddress();
     }
 
+    public function iSpecifyTheBillingSLastNameAs($lastName = null)
+    {
+        $this->addressingPage->specifyBillingAddressLastName($lastName);
+    }
+
+    /**
+     * @When I specify the billing's street as :streetName
+     * @When I do not specify the billing's street
+     */
+    public function iSpecifyTheBillingSStreetAs($streetName = null)
+    {
+        $this->addressingPage->specifyBillingAddressStreet($streetName);
+    }
+
+    /**
+     * @Given I choose :countryName as billing's country
+     */
+    public function iChooseAsBillingSCountry($countryName)
+    {
+        $this->addressingPage->chooseBillingAddressCountry($countryName);
+    }
+
+    /**
+     * @When I specify the billing's city as :cityName
+     * @When I do not specify the billing's city
+     */
+    public function iSpecifyTheBillingSCityAs($cityName = null)
+    {
+        $this->addressingPage->specifyBillingAddressCity($cityName);
+    }
```

Comment:
Mistake during rebase? These steps are removed IIRC and we use full address object.